ethereum/datastructures/src/test/java/tech/pegasys/artemis/datastructures/operations/AttestationDataTest.java

```diff
@@ -170,12 +177,12 @@ void equalsReturnsFalseWhenJustifiedSlotsAreDifferent() {
         new AttestationData(
             slot,
             shard,
-            beaconBlockHash,
-            epochBoundaryHash,
-            shardBlockHash,
-            lastCrosslinkHash,
-            justifiedSlot.plus(randomUnsignedLong()),
-            justifiedBlockHash);
+            beaconBlockRoot,
+            epochBoundaryRoot,
+            shardBlockRoot,
+            latestCrosslink,
+            justifiedEpoch.plus(randomUnsignedLong()),
```

Comment:
Can we rename this test due to justifiedSlots no longer being present? Also it may be beneficial to double check the tests names in this file reflect the actual field being changed.

Reply:
Ah yes!! Thank you for catching!